pytest_docker.py

```diff
@@ -3,13 +3,15 @@
 import sys
```

Comment:
I think these changes should be in separate commit since they are just formatting